app/views/users/new.html.haml

```diff
@@ -0,0 +1,8 @@
+.row
+  = render 'users/left_column'
+
+  .col-6.pt-5.pb-2.pl-5.pr-5
+    - if request.original_fullpath == new_user_registration_path
```

Comment:
I'm wondering if it's better to use `controller_name ` and `action_name` to check which view should be rendered. The reason why I say that is because `original_fullpath` could actually contain additional params as well, and then it might not exactly match `new_user_registration_path`

Reply:
thanks for this suggestion! I didn't know about this methods 🙈 I will use `controller_name` since `action_names` are equal (`new`).